example/laravel56/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM php:latest
+
+RUN apt-get update && \
+  apt-get install -y --no-install-recommends git zip unzip wget
+
+RUN mkdir /app
+WORKDIR /app
+
+COPY . /app/
```

Comment:
everything is relative from the `WORKDIR` once it is set so we can use relative paths here